src/AsmResolver.DotNet/Cloning/CloneContextAwareReferenceImporter.cs

```diff
@@ -5,7 +5,10 @@ namespace AsmResolver.DotNet.Cloning
     /// </summary>
     public class CloneContextAwareReferenceImporter : ReferenceImporter
     {
-        private readonly MemberCloneContext _context;
+        /// <summary>
+        /// The working space for this member cloning procedure.
+        /// </summary>
+        protected readonly MemberCloneContext _context;
```

Comment:
Why was this made `protected`?

Reply:
Only because I thought you would want it to be protected. I didn't have any specific reason on my mind other than that inheritors might need/want access to that field.

Reply:
Implemented